miden-lib/asm/kernels/transaction/lib/prologue.masm

```diff
@@ -315,25 +233,6 @@ proc.validate_new_account
     assert_eqw.err=ERR_PROLOGUE_NEW_ACCT_VAULT_NOT_EMPTY
     # => []
 
-    # Assert storage slot types are well formed
-    # ---------------------------------------------------------------------------------------------
-    # validate storage slot types
-    exec.validate_storage_slot_types
-    # => []
```

Comment:
Not for this PR, but we should probably make sure that all storage offsets for procedures are in-bounds (e.g., no procedure has storage offset greater than the number of storage slots for the account).

Let's create an issue for this.

Reply:
Issue created: https://github.com/0xPolygonMiden/miden-base/issues/863